test/extended/testdata/imagestream-jenkins-slave-pods.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  labels:
+    role: jenkins-slave
```

Comment:
(we should be testing both behaviors:  the default labeling, and the user-provided labeling)